### PR TITLE
CCTRI-1309/tweak documentation

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,122 +1,296 @@
...
+    - `Judgement`,
+    - `Sighting`.
+  - Must be a positive integer. Defaults to `100` (if unset or incorrect).
 
-## Usage
-
-```bash
-pip install -U -r use-requirements.txt
-```
+## Implementation Details
```
Comment (ivlana) [inline README.md:286]: Should tweak the title of this section to be CTIM Mapping Specifics

File: README.md [hunk 2]
```diff
@@ -1,122 +1,296 @@
...
-http POST "${URL}"/health Authorization:"Bearer ${JWT}"
-http POST "${URL}"/deliberate/observables Authorization:"Bearer ${JWT}" < observables.json
-http POST "${URL}"/observe/observables Authorization:"Bearer ${JWT}" < observables.json
-```
\ No newline at end of file
+- `Verdict` is based on `.data.abuseConfidenceScore`
+- `Judgement` is based on `.data.reports[].`
+- `Sighting` is based on `.data.reports[].comment`
+- `Indicator` is based on `.data.reports[].categories[]`
+- `Relationship` has relationship type `sighting-of` which connects
```
Comment (ivlana) [inline README.md:295]: "a" relationship type

File: README.md [hunk 3]
```diff
@@ -1,122 +1,296 @@
...
+the `Sighting` with its `Indicator`
```
Comment (ivlana) [inline README.md:296]: should skip "the"

File: README.md (hunk 1 above)
Comment (GeorgiyII) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (GeorgiyII) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (GeorgiyII) [reply README.md]: done

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
-Flask==1.1.1
-marshmallow==3.4.0
+Authlib==0.14.3
+Flask==1.1.2
+marshmallow==3.6.1
 Werkzeug==1.0.0
-requests==2.23.0
+requests==2.24.0
 beautifulsoup4==4.8.2
+zappa==0.49.0
```
Comment (gdavoian) [inline requirements.txt:7]: Please upgrade to the latest version: 0.51.0

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,7 @@
-Authlib==0.14.1
-Flask==1.1.1
-marshmallow==3.4.0
+Authlib==0.14.3
+Flask==1.1.2
+marshmallow==3.6.1
 Werkzeug==1.0.0
```
Comment (gdavoian) [inline requirements.txt:4]: Do you really need to install this explicitly? It's installed as a Flask's dependency anyway.

File: README.md [hunk 6]
```diff
@@ -1,122 +1,296 @@
...
+
+- `ip`
+- `ipv6`
+
+### JWT Payload Structure
+
+```json
 {
-    "alg": "HS256"
+  "key": "<AbuseIPDB_API_KEY>"
```
Comment (gdavoian) [inline README.md:272]: Upper case again :)

"<ABUSE_IPDB_API_KEY>"

Comment (GeorgiyII) [reply README.md]: done

File: requirements.txt (hunk 5 above)
Comment (GeorgiyII) [reply requirements.txt]: done

File: requirements.txt (hunk 4 above)
Comment (GeorgiyII) [reply requirements.txt]: done

**Pull request merged**